tg_bot/modules/weather.py

```diff
@@ -0,0 +1,70 @@
+import pyowm
+from pyowm import timeutils, exceptions
+
+from typing import Optional, List
+from tg_bot import dispatcher, updater, API_WEATHER
+from tg_bot.modules.disable import DisableAbleCommandHandler
+
+from telegram import Message, Chat, Update, Bot
+from telegram.ext import run_async
+
+@run_async
+def weather(bot: Bot, update: Update, args: List[str]):
+    if len(args) >= 1:
```

Comment:
early returns are preferred to increased indents.
````
if len(args) == 0:
    # reply that a location is needed
    return
# do other stuff
# no else needed, since returned
# less indent = easier to follow
```
This reduces indentation, and makes code easier to read and understand.